structures/withTags.js

```diff
@@ -1,29 +1,18 @@
-const {
-  type,
-  optional,
-  array,
-  enums,
-  refine,
-} = require('superstruct');
+const { type, optional, array, enums, number, refine } = require('superstruct')
+
+exports.withTags = (config = {}) => {
+  const authorizedTags = Array.isArray(config.authorizedTags) ? config.authorizedTags : []
```

Comment:
```suggestion
  const authorizedTags = Array.isArray(config?.authorizedTags) ? config.authorizedTags : []
```
Since nobody prevents you from passing null or undefined as config this minor change could be reasonable

Reply:
Btw, it could be also handy to provide JSDoc here to give an IDE hint to whoever uses this function=)

Reply:
Done